packages/dd-trace/src/debugger/devtools_client/index.js

```diff
@@ -24,11 +25,14 @@ const expression = `
 const threadId = parentThreadId === 0 ? `pid:${process.pid}` : `pid:${process.pid};tid:${parentThreadId}`
 const threadName = parentThreadId === 0 ? 'MainThread' : `WorkerThread:${parentThreadId}`
 
+const oneSecondNs = BigInt(1_000_000_000)
+let globalSnapshotSamplingRateWindowStart = BigInt(0)
```

Comment:
Why not use `1_000_000_000n` instead of `BigInt(1_000_000_000)`? (Same for `0n`.) It's a literal then instead of a function call that converts a double.

Reply:
Ah, I keep forgetting about `n`... If you don't mind, I'll merge this one to keep things moving and make another PR where I "fix" this nit across all files in the code base 😊